ui/StatusQ/CMakeLists.txt

```diff
@@ -168,15 +168,22 @@ add_library(StatusQ SHARED
 
         # onboarding
         src/onboarding/enums.h
+
+        # keychain
+        include/StatusQ/keychain.h
+        src/keychain.cpp
```

Comment:
for `statuswindow` the non-mac impl is defined in `else` branch below. For consistency we could keep the same approach for `keychain`.